docusaurus.config.js

```diff
@@ -3452,7 +3452,25 @@ module.exports = {
           {
             to: '/v2.7/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale',
             from: '/v2.7/reference-guides/best-practices/rancher-server/tips-for-scaling-rancher'
+          },
+          // Redirects for restructure from PR #1147 (start)
+          {
+            to: '/v2.8/api/v3-rancher-api-guide',
+            from: '/v2.8/reference-guides/about-the-api'
+          },
+          {
+            to: '/v2.8/api/api-tokens',
+            from: '/v2.8/reference-guides/about-the-api/api-tokens'
+          },
+          {
+            to: '/api/v3-rancher-api-guide',
+            from: '/reference-guides/about-the-api'
```

Comment:
2064-2067 can be removed and combined into this one.

```suggestion
            from: ['/reference-guides/about-the-api', '/pages-for-subheaders/about-the-api']
```